libraries/wallet/wallet.cpp

```diff
@@ -74,6 +74,9 @@
 #include <graphene/debug_witness/debug_api.hpp>
 #include <fc/smart_ref_impl.hpp>
 
+#include <cstdlib>
+using namespace std;
```

Comment:
Do we still need above lines?

Reply:
ok sure ! sorry ! Thanks ! forgot to remove those lines :-) I think we don't need these lines, but I will check to be 100% sure !